extensions/replication/tasks/CopyLocationTask.js

```diff
@@ -22,21 +22,14 @@ const MPU_GCP_MAX_PARTS = 1024;
 
 class CopyLocationTask extends BackbeatTask {
 
-    _getReplicationEndpointType() {
```

Comment:
nit: to simplify the change, we could have kept this function and (greatly) simplify its implementation


```suggestion
    _getReplicationEndpointType() {
        return this.destConfig.replicationEndpoint.type
    }
```